### PR TITLE
Bengali Tutorial Part One

### PR DESCRIPTION
**Pull request by ucalyptus**
**About Bengali**
 _With approximately 228 million native speakers and another 37 million as second language speakers, Bengali is the fifth most-spoken native language and the seventh most spoken language by total number of speakers in the world._

### CODE REVIEW

**Comment by ucalyptus** [comment]
@karlhigley noted. making changes.

**Comment by ucalyptus** [comment]
@karlhigley changes made, awaiting CI build to finish

**Comment by ucalyptus** [comment]
@karlhigley  Also check https://github.com/OpenMined/PySyft/pull/2942

File: examples/tutorials/translations/bengali/Part 01 - The Basic Tools of Private Deep Learning.ipynb [hunk 1]
```diff
@@ -0,0 +1,642 @@
+{
```
Comment (karlhigley) [inline examples/tutorials/translations/bengali/Part 01 - The Basic Tools of Private Deep Learning.ipynb]: <ul><li>In other languages, we've been putting the translators section below the "Authors" section. I think it should go right under "লেখক: Andrew Trask - Twitter: <a href="https://twitter.com/iamtrask" target="_blank">@iamtrask</a>" </li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/2938/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (ucalyptus) [reply examples/tutorials/translations/bengali/Part 01 - The Basic Tools of Private Deep Learning.ipynb]: ok. missed that. redoing it.

Comment (ucalyptus) [reply examples/tutorials/translations/bengali/Part 01 - The Basic Tools of Private Deep Learning.ipynb]: fixed it

**Comment by ucalyptus** [comment]
@karlhigley changes made. Added the Translator section  right after Authors both here and in #2942

**Comment by karlhigley** [comment]
@ucalyptus Thanks for the translations!

**Comment by ucalyptus** [comment]
@karlhigley Thanks for merging. All of the notebooks to come soon in Bengali

**Pull request merged**